### PR TITLE
Move from Maven to Gradle

### PR DESCRIPTION
**Pull request by wolfieboy09**

### CODE REVIEW

**Comment by wolfieboy09** [comment]
Do not merge or anything until I get my latest commits pushed. Will update once it has been pushed and git lets me push

image

**Review by alex-ritter — CHANGES_REQUESTED** [review]
We need to add gradle dependencies in the .gitignore.

We shouldn't have the .gradle folder checked into source control.

**Review by alex-ritter — APPROVED** [review]
LGTM

File: build.gradle~ [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import org.apache.tools.ant.filters.ReplaceTokens
```
[the remark this answers is not in the document]
Comment (wolfieboy09) [reply build.gradle~:1]: I don't know why its the same file but with a `~` attached, so we can remove it

**Pull request merged**